File: DokanyAPI/DokanyApp.BLL/Entities/Category.cs
using System.Collections.Generic;

namespace DokanyApp.BLL
{
    public partial class Category
    {
        public Category()
        {
            Product = new HashSet<Product>();
        }

        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string ImagePath { get; set; }
        public virtual ICollection<Product> Product { get; set; }
    }
}

File: Dokany UI/src/app/models/Category.ts
export class Category{
    Id: Number;
    Name: String;
    Description: String;
    ImagePath: String

    constructor(_id: Number , _name: String , _description:String ,  _imagePath: String){
        this.Id = _id;
        this.Name = _name;
        this.Description = _description;
        this.ImagePath = _imagePath;
    }
}